File: Programming_Basics_with_Python/conditional_statements_exercise/01._sum_seconds.py
import math
time_one = int(input())
time_two = int(input())
time_three = int(input())
total_time = time_one + time_two + time_three
minutes = total_time / 60
seconds = total_time % 60
minutes = math.floor(minutes)
if seconds < 10:
    print(f"{minutes}:0{seconds}")
else:
    print(f"{minutes}:{seconds}")

File: Programing_Basics_with_JS/conditionalStatementsExercise/01.SumSeconds.js
function sumSeconds(input) {
    let firstTime = Number(input[0]);
    let secondTime = Number(input[1]);
    let thirdTime = Number(input[2]);

    let totalSecondTime = firstTime + secondTime + thirdTime;
    let minutes = Math.floor(totalSecondTime / 60);
    let seconds = totalSecondTime % 60;

    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`);
    } else {
        console.log(`${minutes}:${seconds}`);
    }
}

sumSeconds(["35",
"45",
"44"])

sumSeconds(["22",
"7", 
"34"])

sumSeconds(["50",
"50",
"49"])

sumSeconds(["14",
"12",
"10"])

